### PR TITLE
Optional node for automation hub to be deployed on

### PR DESCRIPTION
**Pull request by cloin**
##### SUMMARY

This just creates a new node with the same image and resources allocated as the tower node.
The node is added to instructor inventories. Not installing Hub at this time.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

- provisioner

### CODE REVIEW

**Comment by IPvSean** [comment]
you need to fix your lint!

```ansible-tox-linters : FAILURE in 5m 45s```

```./provisioner/roles/manage_ec2_instances/tasks/instances/instances_automation_hub.yml
  33:1      error    too many blank lines (1 > 0)  (empty-lines)```

File: provisioner/roles/manage_ec2_instances/tasks/instances/instances_automation_hub.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      long_name: "automation-hub.{{ ec2_name_prefix }}.{{ workshop_dns_zone | default ('') }}"
+    wait: "{{ ec2_wait }}"
+    vpc_subnet_id: "{{ ec2_vpc_subnet_id }}"
+    volumes:
+      - device_name: /dev/sda1
+        volume_type: gp2
+        volume_size: "{{ ec2_info[control_type].disk_space }}"
+        delete_on_termination: true
+  register: hub_output
+
```
Comment (liquidat) [inline provisioner/roles/manage_ec2_instances/tasks/instances/instances_automation_hub.yml:33]: This needs to be removed.

**Comment by IPvSean** [comment]
any updates @cloin

**Comment by cloin** [comment]
recheck.

**Comment by cloin** [comment]
recheck

**Pull request merged**